fastly/resource_fastly_service_v1.go

```diff
@@ -1069,6 +1069,67 @@ func resourceServiceV1() *schema.Resource {
 				},
 			},
 
+			"splunk": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						// Required fields
+						"name": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The unique name of the Splunk logging endpoint",
+						},
+						"url": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The Splunk URL to stream logs to",
+						},
+						"token": {
+							Type:        schema.TypeString,
+							Required:    true,
+							DefaultFunc: schema.EnvDefaultFunc("FASTLY_SPLUNK_TOKEN", ""),
+							Description: "The Splunk token to be used for authentication",
+							Sensitive:   true,
+							StateFunc: func(v interface{}) string {
```

Comment:
For the statefunc we will need to remove this when we merge in the 0.12.x changes/PR later.  For now other resources have these in the 0.11.x release of the provider.